### DISCUSSION TITLE
Deleting users

### DISCUSSION THREAD

**Discussion by adamchen** [fief-dev/fief#156] [Q&A] [answered]
Hi! Hopefully a quick one - how do I delete a user within the cloud version of Fief from the admin panel? I want to delete the user as it's my own email and I'm just doing some testing, but I can't seem to see an obvious way to do it in the UI. Thanks in advance!

**Comment by fief-bailiff** [comment]
Hail, @adamchen 👋 Welcome to Fief's kingdom!

Our team will get back to you very soon to help.

In the meantime, take a minute to star our repository ⭐️

![star-fief](https://user-images.githubusercontent.com/1144727/192523552-dc2c31fa-ae94-42fa-9ab4-7d0ba51bb7b0.png)

Farewell!

**Comment by frankie567** [comment, marked as answer]
Hello!

Unfortunately, we don't have a mechanism to delete users at the moment. I was reluctant to implement it before we had webhooks (— to be sure we have a way to tell apps a user has been deleted —), but now that they're released, we can move forward with this.

Likely, this will be part of the next release!

**Reply by adamchen** [reply]
Hey @frankie567 - thanks for such a quick response. Out of interest, what's the release cadence like, and is there anything I can be helpful with?

**Reply by frankie567** [reply]
I try to release every two weeks!

Well, I'm always looking for feedback about Fief:

* How did it went to integrate with your app?
* Do you miss SDK for specific languages/frameworks?
* What features would like to see?
* Report bugs or problems you may encounter

**Reply by adamchen** [reply]
I actually have very little to complain about - integration was really smooth and I'm looking forward to trying it out a bit more as things on my side get built a bit more! I'm using Python/JS so nothing really missing on that side.

I'll keep an eye out for things and definitely raise if I find anything. Thanks for getting back to me so quickly!

**Reply by frankie567** [reply]
Ability to delete users is now released in [`v0.23.2`](https://github.com/fief-dev/fief/releases/tag/v0.23.2) and available on Fief Cloud 🚀
